### DISCUSSION TITLE
Show the selected country

### DISCUSSION THREAD

**Discussion by fisforfaheem** [Daniel-Ioannou/flutter_country_picker#123] [Q&A]
help wit hthat

**Comment by Daniel-Ioannou** [comment]
@fisforfaheem The package has no option to display the selected country. 
The way I use it, is to add the selected country in the favorites parameter.

``` Dart
Country? selectedCountry;
showCountryPicker(
   context: context,
   onSelect: (Country country) {
      setState(() => selectedCountry = country);
   }, 
   favorite: [
      if(selectedCountry != null) ... [selectedCountry!.countryCode]
   ],
);
```
**In the screenshot below Sweden is the selected country**  
<img height="600" alt="n1" src="https://github.com/Daniel-Ioannou/flutter_country_picker/assets/15341488/87ce3364-51e3-46c3-8fbb-e7280fdb6030">
